### DISCUSSION TITLE
VPN support for coolify and server services

### DISCUSSION THREAD

**Discussion by compgeniuses** [coollabsio/coolify#1847] [Feature Requests]
WOuldnt it be great if we had a vpn service on coolify

**Comment by ItzAmirreza** [comment]
Agreed, OpenVPN is a great choice.

**Reply by honel** [reply]
I would hope that WireGuard is also implemented.

**Reply by aybasaran** [reply]
#rez 4 openvpn

**Comment by diogox** [comment]
Anyone know if this is planned?

Is there a roadmap somewhere I may have missed?

**Comment by dgloukhman** [comment]
what's the usecase? ...you could hook up a gluetun docker container as a vpn client
https://github.com/qdm12/gluetun

**Reply by diogox** [reply]
I haven't tested that approach, but my use case is to be able to access docker container ports that are not exposed to the host.

So if you have a `postgresql` container, you could `postgresql:5432` to access the DB under VPN.

If nothing else, I think one of the issues is there's no way in Coolify to make the VPN have access to all networks.

**Reply by dgloukhman** [reply]
put gluetun on the same network as postgress, than its possible. ...however you have to modify the docker-compose.yml for your deployment for that.
